client/state/selectors/can-site-view-atomic-hosting.js

```diff
@@ -29,13 +30,21 @@ export default function canSiteViewAtomicHosting( state ) {
 	// This is also enforced on the server, please remove client checks later.
 	// ID of site added 31 Oct 2019, so only sites newer currently eligible
 	if ( siteId > 168768859 ) {
+		const canManageOptions = canCurrentUser( state, siteId, 'manage_options' );
+
+		if ( ! canManageOptions ) {
+			return false;
+		}
+
 		const isAtomicSite = !! isSiteAutomatedTransfer( state, siteId );
+		const planSlug = get( getSelectedSite( state ), [ 'plan', 'product_slug' ] );
+		const hasEligablePlan = isBusinessPlan( planSlug ) || isEcommercePlan( planSlug );
 
-		if ( isAtomicSite ) {
+		if ( isAtomicSite && hasEligablePlan ) {
```

Comment:
Do we need the `hasEligiblePlan` check here? Can we have an Atomic site that is not under the Business or eCommerce plan?

Reply:
Confirmed that we only need the site to be Atomic in order to have SFTP and phpMyAdmin working.

I also verified that all sites under an eCommerce plan are Atomic, so we could change the plan check to be only `isBusinessPlan( planSlug )` like before.